### PR TITLE
Fix typos in i18n locales files

### PR DESCRIPTION
**Pull request by Meddiaz**

### CODE REVIEW

File: config/locales/nb.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ nb:
   time_reg:
     errors:
       messages:
-        timer_running: "En annen timer går fortsatt"
+        timer_running: "En annen tidtaker går fortsatt"
```
Comment (hoglund003) [inline config/locales/nb.yml:87]: I think this word is a little strange. How about `stoppeklokke`?

File: config/locales/nb.yml [hunk 2]
```diff
@@ -143,7 +143,7 @@ nb:
     remove_every_task_from_the_project: "Fjern hver oppgave fra prosjektet"
     delete_the_project_permanently: "Slett prosjektet permanent"
     please_enter: "Vennligst skriv inn"
-    delete: "Fjærn"
+    delete: "Fjern"
```
Comment (hoglund003) [inline config/locales/nb.yml:146]: How about `slett`?

File: config/locales/nb.yml [hunk 3]
```diff
@@ -242,7 +242,7 @@ nb:
     with_a_task: "med oppgaven"
     logged_action_cannot_be_undone: "logget. Denne handlingen kan ikke bli tatt tilbake."
     registration_was_successfully: "Tidsregistreringen ble velykket slettet."
-    time_entry_has_been_toggled: "Tids teller har blitt slått"
+    time_entry_has_been_toggled: "Tidsteller har blitt slått"
```
Comment (hoglund003) [inline config/locales/nb.yml:245]: `Stoppeklokke`?

File: config/locales/nb.yml [hunk 4]
```diff
@@ -292,21 +292,21 @@ nb:
...
+    first_name: "Fornavn"
+    last_name: "Etternavn"
+    email: "E-postadresse"
+    password_blank: "passord (la stå tomt for å beholde gammelt passord)"
+    confirm_password: "Bekreft passord (la stå tomt for å beholde gammelt passord)"
     current_password: "Nåværende passord"
     confirm_changes: "Vi trenger ditt nåværende passord for å bekrefte endringene"
     update: "Oppdater"
-    unhappy: "Er du ikke fordnøyed"
+    unhappy: "Er du ikke fornøyd"
```
Comment (hoglund003) [inline config/locales/nb.yml:309]: How about `Er du misfornøyd`?

File: config/locales/nb.yml [hunk 5]
```diff
@@ -414,7 +414,7 @@ nb:
       error: "Feil"
       success: "Suksess"
   time_regs:
-    active_timer_description: "Vær oppmerksom på at du har en timer som går."
+    active_timer_description: "Vær oppmerksom på at du har en tidtaker som går."
```
Comment (hoglund003) [inline config/locales/nb.yml:417]: `stoppeklokke`?

**Review by hoglund003 — CHANGES_REQUESTED** [review]
This is an improvement, but I think some translations are still a bit odd. :sweat_smile:

File: config/locales/nb.yml (hunk 3 above)
Comment (Meddiaz) [reply config/locales/nb.yml:245]: Slått?

File: config/locales/nb.yml [hunk 6]
```diff
@@ -217,16 +217,16 @@ nb:
     task_is_already_assigned: "Oppgaven er allerede tildelt prosjekter, kan ikke slettes."
     the_client_has_associated_projects: "Kunden har tilknyttede prosjekter, kan ikke slettes."
     add_task_to: "Legg oppgave til"
-    optional_project: "Dette er valgfritt, ellers bør prosjekt pris bli brukt"
-    add: "Leg Till"
+    optional_project: "Dette er valgfritt, ellers bør prosjektpris bli brukt"
+    add: "Legg Til"
     looking_for: "Kan du ikke finne hva du leter etter?"
     update: "Oppdater"
     project_rate: "prosjekt Verdi"
```
Comment (Meddiaz) [inline config/locales/nb.yml:224]: Prosjektrate? Sats? Timepris?

**Review by hoglund003 — APPROVED** [review]
LGTM

**Pull request merged**